backend/S3/app.py

```diff
@@ -39,33 +40,36 @@ def create(key, fileLocation):
 
 
 def read(key):
-    PresignedUrl = user.generate_presigned_url(
-        'get_object', Params={'Bucket': bucketLocation, 'Key': key}, ExpiresIn=3600)
-    if not fileExist(key):
-        raise FileNotFoundError
-    else:
-        return PresignedUrl
-
-# update
-
-
-def update(oldKey, newKey, fileLocation=''):
     try:
```

Comment:
A try-except block has been added around the ```read()``` code of ```S3.app```, otherwise an error with empty error message will be thrown when the function is used in my code of processing GET request at ```/social/posts``` (no arguments) endpoint.